single_tech_samples/fabric/feature_engineering_on_fabric/README.md

```diff
@@ -379,6 +400,40 @@ If the pipeline executes successfully, you can verify the features registered in
 
   ![feature_lineage](./images/managed_feature_store.gif)
 
+### Exploratory data analysis (EDA)
+
```

Comment:
Ordering of the content. Suggest, first giving a brief overview of what EDA is and then add the notebook specific detail.

Also, provide link to the notebook at-least once. i.e, 

replace `exploratory_data_analysis` with [exploratory_data_analysis.ipynb](./src/notebooks/exploratory_data_analysis.ipynb)

And do the same for data validation step.

Reply:
Hi @promisinganuj, 
Thanks for your great suggestions as always! 
Made some updates to address several of them, thanks for your help to check again! :- )